synapse/federation/federation_client.py

```diff
@@ -467,13 +472,24 @@ async def get_room_state_ids(
         state_event_ids = result["pdu_ids"]
         auth_event_ids = result.get("auth_chain_ids", [])
 
+        set_tag(
+            SynapseTags.RESULT_PREFIX + f"state_event_ids ({len(state_event_ids)})",
+            str(state_event_ids),
+        )
```

Comment:
Won't spaces and a varying number in the tag name make it hard to search by tag in jaeger?

Reply:
I guess so. Do we need to search by these result tags?

Reply:
It's hard to say in advance. I've used the tags to find slow requests to certain rooms in the past.
I'm in favor of moving the count to its own tag if we find it useful.

Reply:
I've separated out the length ⏩